dashboard/config/routes.rb

```diff
@@ -317,6 +317,13 @@ def redirect_to_teacher_dashboard
 
   post '/plc/course_units/:id/submit_new_questions_and_answers', to: 'plc/course_units#submit_new_questions_and_answers'
 
+  namespace :pd do
+    get 'workshops/:workshop_id/enroll', action: 'new', controller: 'workshop_enrollment'
+    post 'workshops/:workshop_id/enroll', action: 'create', controller: 'workshop_enrollment'
+    get 'workshop_enrollment/:code', action: 'show', controller: 'workshop_enrollment'
```

Comment:
What's the difference between workshops/X/enroll and workshop_enrollment/C?  Should they both be under /workshops?